api/src/main/java/org/pmiops/workbench/notebooks/NotebooksServiceImpl.java

```diff
@@ -342,7 +342,8 @@ public String getReadOnlyHtml(
   @Override
   public String adminGetReadOnlyHtml(
       String workspaceNamespace, String workspaceName, String notebookNameWithFileExtension) {
-    if (!NotebookUtils.isJupyterNotebook(notebookNameWithFileExtension)) {
+    if (!(NotebookUtils.isJupyterNotebook(notebookNameWithFileExtension)
```

Comment:
Is this addition necessary? 

How I read this, it is saying:
If not a Jupyter notebook or if it is an RStudio file, throw a Not Implemented Exception.

I would suspect that that all RStudio files are not Jupyter notebooks.

Reply:
Never mind, I misread where the parentheses were. This looks good!